Source/ProgressionSystemRuntime/Public/Widgets/PSEndGameWidget.h

```diff
@@ -4,13 +4,13 @@
 
 #include "Blueprint/UserWidget.h"
 #include "PoolManagerTypes.h"
-#include "PSMenuWidget.generated.h"
+#include "PSEndGameWidget.generated.h"
 
 /**
  * Widget to display the progression as stars in the main menu.
```

Comment:
`in the main menu` - please update description

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/43/commits/f48a0da673c04aea830128a3ab9d506b1f4858c7